src/main/kotlin/blackjack/model/Dealer.kt

```diff
@@ -0,0 +1,15 @@
+package blackjack.model
+
+class Dealer(
+    override val name: String = "Dealer",
+    override var isActive: Boolean = true,
```

Comment:
What does `isActive` mean? 🤔 
I think it would be more intuitive to use actual blackjack rules!